vagrant/Vagrantfile

```diff
@@ -13,7 +13,8 @@ Vagrant.configure("2") do |config|
     sudo apt-get install -yq build-essential libffi-dev redis git-core software-properties-common
     sudo add-apt-repository ppa:deadsnakes/ppa
     sudo apt-get update
-    sudo apt-get install python3.6 -y
+    sudo apt-get install python3.8-dev -y
+    sudo apt-get install python3.8 -y
```

Comment:
Did the vagrant worked with `cpu>1` for `python3.8`?

Reply:
No, the cpu value had to be changed to `1`. I did not commit that change because I thought it had to do with my system rather than with the PR content. Should I add those changes instead?

Reply:
Umm, leave number of cpus unchanged but do leave an inline comment to change it to cpu=1 in case this Vagrantfile didn't worked in anyone's machine.

Reply:
I pushed a commit adding an inline comment. Thanks for spotting this!